### ISSUE TITLE
Plotting just exons

### ISSUE THREAD

**Issue by oalavijeh** [jianhong/trackViewer#16]
Dear Jianhong,

Many thanks for this great package. I have a very long gene which I only wanted to plot the exons of to enable the graph to be pulled out more. and more informative. At present it is very compressed as per the below diagram. Would that be possible?

Many thanks

![image](https://user-images.githubusercontent.com/56344383/126626309-fa9fa9d9-86b3-4e4d-a0fd-76b1ae7faf4d.png)

**Comment by jianhong** [comment]
Hi,

Maybe this section may help https://bioconductor.org/packages/release/bioc/vignettes/trackViewer/inst/doc/lollipopPlot.html#Change_the_scale_of_the_x-axis_(xscale)

Currently we don't have a quick setting for only plot exon regions.

Jianhong.

**Comment by jianhong** [comment]
I just updated the development version of trackViewer to make it possible by a quick setting.
Try:

```
BiocManager::install("jianhong/trackViewer")
packageVersion("trackViewer") #1.29.2
detach("package:trackViewer", unload=TRUE)
library(trackViewer)
lolliplot(snps, features, ranges=gr, rescale = "exon")
lolliplot(snps, features, ranges=gr, rescale = "exon_95") #95%
```
Jianhong.

**Comment by oalavijeh** [comment]
Thanks for implementing this! At the moment I am getting the following error:

error in lolliplot(sample.gr, features, ranges = gr, rescale = "exon") : 
  Something wrong with the auto-scale setting. Please report the bug to https://github.com/jianhong/trackViewer/issues

**Comment by jianhong** [comment]
Do you mind to send me the minimal rda files to repeat the error?

**Comment by oalavijeh** [comment]
really sorry but what command would I use to generate that?

**Comment by oalavijeh** [comment]
I figured it out, but is there a way of sharing this with you in private please?

**Comment by jianhong** [comment]
It should be fix in version 1.29.4. Let me know if you still have question.

**Comment by oalavijeh** [comment]
Dear @jianhong ,

I have jus tried again but got the same error as before. May I ask how you installed the latest version?

All the best
